mava/wrappers/gym.py

```diff
@@ -196,7 +196,7 @@ def modify_space(self, space: spaces.Space) -> spaces.Space:
 
 
 class GymToJumanji(gymnasium.Wrapper):
-    """Converts Gym outputs to Jumanji timesteps"""
+    """Converts from the Gym API to the dm_env API, Jumanji's Timestep type."""
```

Comment:
Sorry this might have been a typo on my end, just grammar
```suggestion
    """Converts from the Gym API to the dm_env API, using Jumanji's Timestep type."""
```